lib/cloud_controller/deployment_updater/updater.rb

```diff
@@ -84,7 +84,7 @@ def scale_deployment
           end
 
           scale_down_oldest_web_process_with_instances
-          deploying_web_process.update(instances: deploying_web_process.instances + 1)
+          deploying_web_process.update(instances: deploying_web_process.instances + deployment.max_in_flight)
```

Comment:
in cases where total number of instances is not divisible by `max_in_flight`, will this result in extra instances ?  Should there be a check like this?

```
[deploying_web_process.instances + deployment.max_in_flight, deployment.original_web_process_instance_count].min
```